daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -88,7 +89,11 @@ func init() {
 // the EC2 Instance Metadata Service.
 func New(ctx logger.Context) (logger.Logger, error) {
 	logGroupName := ctx.Config[logGroupKey]
-	logStreamName := ctx.ContainerID
+	logStreamName, err := loggerutils.ParseLogTag(ctx, loggerutils.DefaultTemplate)
```

Comment:
I just looked at `loggerutils.DefaultTemplate` and it's `{{.ID}}`, which would be a change in behavior from the existing driver.  Can you use `{{.FullID}}` instead to keep the existing behavior?